kolibri/plugins/device/assets/src/views/FacilitiesPage/CreateNewFacilityModal.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+
+  <KModal
+    :title="$tr('createFacilityLabel')"
+    :submitText="$tr('createFacilityButtonLabel')"
+    :cancelText="coreString('closeAction')"
+    size="medium"
+    @submit="handleSubmit"
+    @cancel="$emit('cancel')"
+  >
+    <p>{{ coreString('learningFacilityDescription') }}</p>
+    <KTextbox
+      ref="facilityNameTextBox"
+      v-model="facilityName"
+      :label="$tr('facilityNameFieldLabel')"
+      :invalid="facilityNameInvalid"
+      :invalidText="coreString('requiredFieldError')"
+      :maxlength="50"
+    />
+    <b>{{ $tr('learningEnvironmentHeader') }}</b>
+    <KRadioButton
+      v-model="preset"
+      class="permission-preset-radio-button"
+      :value="Presets.NONFORMAL"
+      :label="$tr('nonFormalLabel')"
+      :description="$tr('nonFormalDescription')"
+    />
+    <KRadioButton
+      v-model="preset"
+      class="permission-preset-radio-button"
+      :value="Presets.FORMAL"
+      :label="$tr('formalLabel')"
+      :description="$tr('formalDescription')"
+    />
+  </KModal>
+
+</template>
+
+
+<script>
+
+  import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+  import commonSyncElements from 'kolibri.coreVue.mixins.commonSyncElements';
+  import { Presets } from 'kolibri.coreVue.vuex.constants';
+  import { createFacility } from './api';
+
+  export default {
+    name: 'CreateNewFacilityModal',
+    mixins: [commonCoreStrings, commonSyncElements],
+    data() {
+      return {
+        facilityName: '',
+        preset: 'nonformal',
```

Comment:
We probably want to now use the `Presets.NONFORMAL` here instead of hardcoded `nonformal` string.